### ISSUE TITLE
Provision Microsoft 365 learning pathways template failed

### ISSUE THREAD

**Issue by pgdrews** [SharePoint/sp-provisioning-service#366]
**Describe the issue**
Your request to provision "Microsoft 365 learning pathways" template failed!
Please report the issue here providing the following provisioning action correlation ID: 24154e67-5dae-401c-afeb-8e0c78336926 !
**Expected behavior**
I was starting from scratch to deploy LP's.  Per Julie Turner, I deleted the prior M365LP's and then deleted the LP app (customlearning pkg).  Then, redeploy LP from Lookbook provisioning and it failed. 

**Screenshots**
![image](https://user-images.githubusercontent.com/7563343/102911035-1fae1f00-4441-11eb-99dc-08b866095b0c.png)

![image](https://user-images.githubusercontent.com/7563343/102911087-36ed0c80-4441-11eb-9601-c950f01a6aee.png)
![image](https://user-images.githubusercontent.com/7563343/102911128-43716500-4441-11eb-8fce-59dacd222181.png)

**Details (please complete the following information):**
- PnP Correlation ID: 24154e67-5dae-401c-afeb-8e0c78336926
- Reference Template: https://lookbook.microsoft.com/details/3df8bd55-b872-4c9d-88e3-6b2f05344239
- Primary language of the target tenant: English
- Are you using a Tenant Global Admin account to provision the template: YES

**Additional context**
I was starting from scratch to deploy LP's.  Per Julie Turner, I deleted the prior M365LP's and then deleted the LP app (customlearning pkg).  Then, redeploy LP from Lookbook provisioning and it failed.

**Comment by PaoloPia** [comment]
Hi @pgdrews,
Thanks for using the Look Book site. Looking into the events log, I can see that the exception is related to the app in the App Catalog (FileNotFoundException). It looks like the provisioning engine couldn't find the app package and wasn't able to install it on the target site. This is, indeed, a wrong behavior because I can clearly see that you have the SPPKG file in your App Catalog, based on the above screenshot. I'm wondering what the reason could be. Is the user account that you used to provision LP the same that you used to look into the App Catalog and take the screenshot? Anything else that comes into your mind about permissions for your App Catalog? Thanks.

**Comment by pgdrews** [comment]
@PaoloPia appreciate the follow up.  

Well, I deleted and re-provisioned several times and still had no luck, until I changed the URL of the Learning Pathways site.  Then, I proceeded to deploy the SharePoint Success Site (which I also had to use a new URL) and got them both to report as successful.  Learning Pathways appears to be working correctly, but when I open Success Site, there is an error on the administration page. 

![image](https://user-images.githubusercontent.com/7563343/103707217-e19e1880-4f73-11eb-81e9-c868363a6e90.png)

When I use the console logs in the browser for errors, this is what I see.   Any thoughts on on how to resolve?

![image](https://user-images.githubusercontent.com/7563343/103707332-072b2200-4f74-11eb-9fc6-cd9d9fd1d3eb.png)

**Comment by juliemturner** [comment]
I suspect the issue you're having now is that you didn't complete the SharePoint Success Site provisioning steps as outlined in the templates provisioning instructions. I suspect if you follow [these directions](https://docs.microsoft.com/en-us/sharepoint/provision-sss-lookbook) starting at step 6 that basically will add the content pack to learning pathways you should be all set. -- If you prefer video you can watch the provisioning process [here](https://www.youtube.com/watch?v=HZjxBAKVnJs&feature=youtu.be).

**Comment by pgdrews** [comment]
Excellent - that resolved!  Thanks Julie!  

I see that if I had watched the VIDEO link for "Before you start", I would have caught that.  I'm not as much of a video watcher most times.   And, I took the assumption it was just walk through of what I already had done many times with all the other PnP templates...but clearly there was a critical step missed. 

Is it possible to be more literal right after that video line to point to the directions link?  I see it is down below on the page as well under resources.  But nothing stood out that there was some required post steps.  

Thanks again!!!

![image](https://user-images.githubusercontent.com/7563343/103907474-ef17e780-50c6-11eb-9df7-6ae3aa8bb156.png)
